content/SCALE/SCALETutorials/Apps/StableApps/SyncthingStable.md

```diff
@@ -0,0 +1,187 @@
+
+{{< trueimage src="/images/SCALE/Apps/InstallSyncthingStorageConfigData1HostPath.png" alt="Syncthing Data1 Storage Settings" id="Syncthing Data1 Storage Settings" >}}
+
+Enter or browse to the dataset created to populate the **Host Path** field (for example, */mnt/tank/syncthing/config*), then enter or browse to the location of the **data1** dataset to populate the **Host Path** field under the **Mount Path** field.
+
+To add another dataset path inside the container, see [**Storage Settings**](#storage-settings) below for more information.
+Set **Type** to **Host Path (Path that already exists on the system)**, enter **/data1** in **Mount Path**, and then either enter or browse to the path to the **data1** dataset to populate the **Host Path** field.
+
+Accept the default port numbers in **Networking**. 
+See [Network Settings](#networking-settings) below for more information on network settings. 
+Before changing the default port number, see [Default Ports](https://www.truenas.com/docs/references/defaultports/) for a list of assigned port numbers.
+When selected, **Host Network** binds to the default host settings programmed for Syncthing.
+When enabled the app uses the host network for Syncthing.
+The TCP and UDP ports listen on port **20978** and **20979**.
+Web UI listens on port **20910**.
+
+Syncthing does not require advanced DNS options. If you want to add DNS options, see [Advanced DNS Settings](#advanced-dns-settings) below.
+
+Accept the default resource limit values for CPU and memory or select **Enable Pod resource limits** to show the CPU and memory limit fields, then enter the new values you want to use for Syncthing. See [Resource Configuration Settings](#resource-configuration-settings) below for more information.
+
+Click **Install**. 
+The system opens the **Installed Applications** screen with the Syncthing app in the **Deploying** state.
+After installation completes the status changes to **Running**. 
+
+{{< trueimage src="/images/SCALE/Apps/SyncthingChartsInstalled.png" alt="Syncthing Installed" id="Syncthing Installed" >}}
+
+Click **Web Portal** on the **Application Info** widget to open the Syncthing web portal to begin configuring folders, devices, and other settings.
+
+{{< trueimage src="/images/SCALE/Apps/SyncthingWebPortalForTrueNAS.png" alt="Syncthing Web Portal for TrueNAS" id="Syncthing Web Portal for TrueNaS" >}}
+
+### Securing the Syncthing Web UI
+After installing and starting the Syncthing application, launch the Syncthing web UI.
+Go to **Actions > Settings** and set a user password for the web UI.
+
+{{< trueimage src="/images/SCALE/Apps/SyncthingUIActionsMenu.png" alt="Syncthing UI Actions Menu" id="Syncthing UI Actions Menu" >}}
+
+### Using the Syncthing Web Portal for TrueNAS
+
+{{< include file="/static/includes/SyncthingWebPortalInfo.md" >}}
+
+## Understanding TrueNAS Syncthing Wizard Settings
```

Comment:
Some of these details seem outdated in EE as well, for example I get "Network Configuration" settings after "User and Group Configuration" instead of in a "Networking Settings" section toward the end, but I don't know if you were intentionlly avoiding digging into the minute details at this point?

Reply:
I did update with what is in EE, but maybe they made a few more changes in the last INTERNAL build. I'll update and verify.